DNN Platform/Library/Entities/Modules/ModuleController.cs

```diff
@@ -250,6 +251,90 @@ private void CopyTabModuleSettingsInternal(ModuleInfo fromModule, ModuleInfo toM
             }
         }
 
+        /// <summary>
+        /// Checks whether module VIEW permission is inherited from its tab
+        /// </summary> 
+        /// <param name="module">The module</param>
+        /// <param name="permission">The module permission.</param>
+        private bool IsModuleViewPermissionInherited(ModuleInfo module, ModulePermissionInfo permission)
+        {
+            Requires.NotNull(module);
+
+            Requires.NotNull(permission);
+
+            var permissionViewKey = "VIEW";
+
+            if (!module.InheritViewPermissions || permission.PermissionKey != permissionViewKey)
+            {
+                return false;
+            }
+
+            var tabPermissions = TabPermissionController.GetTabPermissions(module.TabID, module.PortalID);
+            
+            return tabPermissions?.Where(x => x.RoleID == permission.RoleID && x.PermissionKey == permissionViewKey).Any() == true;

```

Comment:
You can simplify this as in the following:

return tabPermissions?.Any(x => x.RoleID == permission.RoleID && x.PermissionKey == permissionViewKey) == true;

Reply:
No, we can't use `Any`. This is already mentioned above
https://github.com/dnnsoftware/Dnn.Platform/pull/3132#discussion_r333040908